examples/standalone-server.js

```diff
@@ -52,7 +52,9 @@ const startServer = (port, auth) => http.createServer((req, res) => {
 
 (async () => {
   try {
-
+    const NGROK_AUTH_TOKEN = process.env.NGROK_AUTH_TOKEN
+    if(NGROK_AUTH_TOKEN)
+    await ngrok.authtoken(process.env.NGROK_AUTH_TOKEN)
```

Comment:
Style: indent and spacing. Add a space between `if` and its bracket; surround the `if` condition in square brackets; add a `;` at the end of the statement:

```js
if (NGROK_AUTH_TOKEN) {
  await ngrok.authtoken(process.env.NGROK_AUTH_TOKEN);
}
```